source/connect/connection-targets.txt

```diff
@@ -111,6 +111,10 @@ set ``myreplset``, which is running on port ``27017`` of three different hosts
    of its ``nodes`` attribute, the list might be empty while the client connects to
    the replica-set members.
 
+.. note:: Replica Set in Docker
```

Comment:
Since this is resulting in stacked note boxes which we should avoid, can the paragraph above this be taken out of an admonition? I think it works just as well without being in a note box